modules/lfs/content_store.go

```diff
@@ -53,13 +53,21 @@ func (s *ContentStore) Get(meta *models.LFSMetaObject, fromByte int64) (io.ReadC
 	}
 	if fromByte > 0 {
 		if fromByte >= meta.Size {
+			err = f.Close()
+			if err != nil {
+				log.Error("Whilst trying to read LFS OID[%s]: Unable to close Error: %v", meta.Oid, err)
+			}
 			return nil, ErrRangeNotSatisfiable{
 				FromByte: fromByte,
 			}
 		}
 		_, err = f.Seek(fromByte, io.SeekStart)
 		if err != nil {
 			log.Error("Whilst trying to read LFS OID[%s]: Unable to seek to %d Error: %v", meta.Oid, fromByte, err)
+			errClose := f.Close()
```

Comment:
If we close the `f` here, but we don't return immediately. but we still return `f` out of the function.